content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -152,8 +152,8 @@ You can see examples of the proposed snapshot and release versions in your case
 +
 [source,shell]
 ----
-mvn validate
-mvn validate -Dset.changelist -Dignore.dirt
+mvn help:evaluate -Dexpression=project.version
+mvn help:evaluate -Dexpression=project.version -Dset.changelist -Dignore.dirt
```

Comment:
Actually I think leaving this as `mvn validate` is better, as it runs some additional checks which would help catch some kinds of mistakes. This is a one-time operation so a few extra seconds hardly matters.

Reply:
Isn't `validate` part of basically any actual build?

Reply:
Yes of course, but may as well check basic stuff before pushing a commit.

Reply:
The benefit of the explicit `help:evaluate` is that it makes it easier to read the effective version, as it's not just a side effect of having lines like

```
[INFO] ----------< org.jenkins-ci.plugins:antisamy-markup-formatter >----------
[INFO] Building OWASP Markup Formatter Plugin 155.v795fb_8702324
[INFO] --------------------------------[ hpi ]---------------------------------
```

WDYT about `mvn validate help:evaluate -Dexpression=project.version …`? That way, towards the end of the build log, there would still be a line only containing the version number.

Reply:
Sure.